src/vis/sidebar/AggregateTypeSelect.tsx

```diff
@@ -13,16 +12,6 @@ interface AggregateTypeSelectProps {
   currentSelected: EAggregateTypes;
 }
 
-function formatOptionLabel(option, ctx) {
-  return (
-    <>
-      <Highlighter searchWords={[ctx.inputValue]} autoEscape textToHighlight={option.name} />
-      {option.description && <span className="small text-muted ms-1">{option.description}</span>}
-      {option.disabled ? <i className="ms-1 fas fa-question-circle" title="No numerical columns available for this aggregation type" /> : null}
-    </>
-  );
-}
```

Comment:
the format option label moved into the sidebar/utils.tsx, so that I didnt duplicate it in every one of the react-select components. This one was just a leftover that wasnt being used anyways.

Reply:
Sounds good.